test/test_endpoints.py

```diff
@@ -430,17 +430,20 @@ def test_endpoint_connect_tor(self, private_tor, global_tor, system_tor):
 class FakeProtocol(object):
     implements(IProtocol)
 
-    def dataReceived(self, data):
+    @staticmethod
+    def dataReceived(data):
         print "DATA", data
 
-    def connectionLost(self, reason):
+    @staticmethod
+    def connectionLost(reason):
         print "LOST", reason
```

Comment:
I know these methods don't use self but defining them as @staticmethod kind of goes against the interface definition...not entirely sold ;)

However I am interested in "more PEP8" in general :)